lib/report_runner.rb

```diff
@@ -34,14 +36,28 @@ def generate_report
       ga_profile_id: @ga_profile_id ,
       start_date: @start_date,
       end_date: @end_date,
-      output: @output
+      output: @output,
+      google_parent_id: @google_parent_id
     })
 
     report_generator.generate_report!
   end
 
 private
 
+  def has_google_parent_id?
```

Comment:
This name doesn't imply that this is only needed if `output` is `google-sheets`. Given the name, I'd expect it to fail if `google_parent_id` is missing, whether `output` is `google-sheets` or not.

Maybe `check_google_parent_id?`? That's got a bit more wiggle room. 

`has_google_parent_id_if_needed?` or `needs_and_has_google_parent_id?` is too clunky I feel.

Reply:
I agree - changes in 98164f08086d416c3ca1b6a3b483ed9056c840c2.